test/blocks/quiz-entry/quiz-entry.test.js

```diff
@@ -156,4 +156,147 @@ describe('Quiz Entry Component', () => {
     await new Promise((resolve) => setTimeout(resolve, 100));
     expect(continueButton.classList.contains('disabled')).to.be.false;
   });
+  it('should navigate the carousel using keyboard commands', async () => {
+    const options = document.querySelectorAll('.quiz-option');
+    const option = document.querySelector('.quiz-option');
+    option.click();
+    await new Promise((resolve) => setTimeout(resolve, 100));
+    const carousel = document.querySelector('.quiz-options-container');
+    const rightArrowEvent = new KeyboardEvent('keydown', { key: 'ArrowRight' });
+    const leftArrowEvent = new KeyboardEvent('keydown', { key: 'ArrowLeft' });
+    carousel.dispatchEvent(rightArrowEvent);
+    await new Promise((resolve) => setTimeout(resolve, 100));
+    carousel.dispatchEvent(leftArrowEvent);
+    await new Promise((resolve) => setTimeout(resolve, 100));
+    const leftArrow = document.querySelector('.carousel-arrow.arrow-prev');
+    expect(leftArrow).to.not.exist;
+
+    const tabKeyEvent = new KeyboardEvent('keydown', { key: 'Tab' });
+    option.dispatchEvent(tabKeyEvent);
```

Comment:
Nit: Same here, we can skip caching. There are many optimising opportunities like this in the file, I won't add a comment for all of them. This is just a possible improvement to the code compactness, nothing critical.
```suggestion
    option.dispatchEvent(new KeyboardEvent('keydown', { key: 'Tab' }));
```